### ISSUE TITLE
Snap Links removed from Chrome Web Store

### ISSUE THREAD

**Issue by smaragdus** [cpriest/SnapLinksPlus#309]
[Snap Links](https://cpriest.github.io/SnapLinksPlus/#/) is no longer available on [Chrome Web Store](https://chrome.google.com/webstore/detail/snap-links/ikglmligndmabebhnicldebpekldnabm), Any idea why the extension has been removed?

**Comment by cpriest** [comment]
It was removed by Google because, purportedly, it does not comply with their latest rules for free software.

**Comment by smaragdus** [comment]
Lately I see well-established free extensions being removed from the store, for example this happened to [Stylus](https://github.com/openstyles/stylus/issues/857). It seems that communication with Google is hard (this [comment](https://github.com/openstyles/stylus/issues/857#issuecomment-592569691) is a good example). Anyway, I hope that Snap Links will be available on Chrome Web Store soon.

**Comment by cpriest** [comment]
Unfortunately, it's compounded by the fact that small Open Source / Free software projects rarely have the human resources to deal with the demand when they gain popularity.

I knew this was coming, about 3 days in advance.  I'm lucky if I have an hour a week I could spend on the project let alone with 3 days notice.

**Comment by lucknaumann** [comment]
Thanks for this extension and your work!
Could you put it with the releases? I see one for Firefox. 
I know the dev (gorhill) of uBlock does this. I think mostly for the above mentioned reasons and partly so if users want to download the extension before it hits the store (including for FF), they can. He usually leaves a message about CWS (I'm not saying you need to or should). This is his latest one for stable release:

> [It may take many weeks for submitted stable releases to become available in the CWS.](https://github.com/gorhill/uBlock/releases/tag/1.26.0)

**Comment by cpriest** [comment]
I can't promise I'll be
      able to do so, but I'll give it a shot.
    
    On 4/18/2020 12:39 AM, lucknaumann
      wrote:
    
    
      
      Thanks for this extension and your work!
        Could you put it with the releases? I see one for Firefox.
        I know the dev (gorhill) of uBlock does this. I think mostly for
        the above mentioned reasons and partly so if users want to
        download the extension before it hits the store (including for
        FF), they can. He usually leaves a message about CWS (I'm not
        saying you need to or should). This is his latest one for stable
        release:
      
        It may take many weeks for submitted
            stable releases to become available in the CWS.
      
      —
        You are receiving this because you were assigned.
        Reply to this email directly, view it on GitHub, or unsubscribe.
      [
{
"@context": "http://schema.org",
"@type": "EmailMessage",
"potentialAction": {
"@type": "ViewAction",
"target": "https://github.com/cpriest/SnapLinksPlus/issues/309#issuecomment-615573406",
"url": "https://github.com/cpriest/SnapLinksPlus/issues/309#issuecomment-615573406",
"name": "View Issue"
},
"description": "View this Issue on GitHub",
"publisher": {
"@type": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]

**Comment by lucknaumann** [comment]
I understand. It would be great if you could. Thanks! =)

**Comment by benoit-kohler** [comment]
Hello. Could you please provide a crx release or unpacked (developer mode) for Chrome that we can install in the meantime ?

**Comment by Taps7734** [comment]
Or would it be possible to add it to the MS Store for MS Edge Chromium.  I Am using in the latest DEV and RELEASE builds and it appears to be working well for me using developer mode.

**Comment by cpriest** [comment]
It's possible, it depends on what kind of automation they have available for submission.

**Comment by leodoi3** [comment]
Could you recommend an alternate for chrome?

**Comment by benoit-kohler** [comment]
> Could you recommend an alternate for chrome?

Linkclump

**Comment by zotabee** [comment]
Switched to Linkclump too 👌

**Comment by chaoscreater** [comment]
It has been over a year since OP posted this and still no source code released for Chrome.

I'm super confused. Clearly, source code for SnapLinksPlus-3.1.1b1 was already released for Chrome, then we stopped getting the source code for newer releases afterwards. What has changed?

I don't quite understand why we keep getting source code for Firefox releases but not Chrome. Isn't it just a simple matter of uploading it to Github?

**Comment by remi-garcia** [comment]
I don't use Chrome and I strongly believe everyone should avoid it so I won't do much work to make it simpler. However there is a zip file that it should be possible to use for adding SLP3 as an add-on for Google Chrome.
[slp3.zip](https://github.com/cpriest/SnapLinksPlus/files/7806560/slp3.zip)

I have no clue on how it works but I built that from the dev branch today.

**Comment by chaoscreater** [comment]
@remi-garcia - thanks, works great in Edge!

**Comment by cpriest** [comment]
Yeah, @remi-garcia comment is essentially the reason I have not made it available for chrome.  They've become the Microsoft Internet Explorer of the modern age.  

For a period of time, I had given up on FF ever being fast again (and I was a hard-core believer forever).  At any rate, I switched back again, quite some time ago, since they got their house in order and also the aforementioned problem of being the enormously dominant browser I just haven't given it any further attempt.

Essentially they took it down without notice of curing the problem and without any clear indication as to why other than "it violates our new 14-page free extension policy (also a symptom of a problematic dominant monopoly)."  I don't really have the time for that kind of crap, so I said forget it.

**Comment by leodoi3** [comment]
I’m using Brave and I have the LinkClub extension.  Essentially the only thing it doesn’t have is the ability to select checkboxes and radio buttons which I miss from SnapLinks. I hope you’ll consider releasing a version for Chromium browsers who don’t use Google chrome.

**Comment by chaoscreater** [comment]
> Yeah, @remi-garcia comment is essentially the reason I have not made it available for chrome. They've become the Microsoft Internet Explorer of the modern age.
> 
> For a period of time, I had given up on FF ever being fast again (and I was a hard-core believer forever). At any rate, I switched back again, quite some time ago, since they got their house in order and also the aforementioned problem of being the enormously dominant browser I just haven't given it any further attempt.
> 
> Essentially they took it down without notice of curing the problem and without any clear indication as to why other than "it violates our new 14-page free extension policy (also a symptom of a problematic dominant monopoly)." I don't really have the time for that kind of crap, so I said forget it.

I personally dislike Chrome and can understand where you're coming from. However, uploading the source code to GitHub and uploading it to Chrome Web Store aren't the same thing? There are plenty other Chromium based browsers that are still good. I personally use Firefox, Vivaldi and Edge and they each have their own unique features and extensions. By publishing the source code for Chromium browsers, people at least have the option to sideload it.

> I’m using Brave and I have the LinkClub extension. Essentially the only thing it doesn’t have is the ability to select checkboxes and radio buttons which I miss from SnapLinks. I hope you’ll consider releasing a version for Chromium browsers who don’t use Google chrome.

I got it working for Edge by sideloading it. Check the link posted by remi-garcia above. The old version 3.1.4 in the Releases section also works, but outdated of course.

**Comment by remi-garcia** [comment]
We should close this issue and provide an answer to that in a FAQ

**Comment by RAINVILLEMAX** [comment]
@remi-garcia Thanks

**Comment by jesus2099** [comment]
[Homepage](https://cpriest.github.io/SnapLinksPlus/) Chrome Web Store link should be changed to a FAQ, to this issue, or to [the side load zip file](https://github.com/cpriest/SnapLinksPlus/issues/309#issuecomment-1004643773), specifically.
